src/comfy_script/runtime/factory.py

```diff
@@ -338,6 +338,9 @@ def type_and_hint(type_info: str | list[str | bool], name: str = None, optional:
             if c is None:
                 c = t.__name__
 
+            if isinstance(t, type(StrEnum)):
```

Comment:
```suggestion
            if isinstance(t, type(astutil.StrEnum)):
```
`enum.StrEnum` requires Python 3.11. To support Python 3.9~3.10, the runtime will use a custom `StrEnum` under them.